docs/src/pages/GettingStartedPage/GettingStartedPage.js

```diff
@@ -31,8 +31,8 @@ import Code from '../../components/Code';
 import CodeBlock from '../../components/CodeBlock';
 
 /* eslint-disable import/no-webpack-loader-syntax */
+/* eslint-disable import/no-unresolved */
```

Comment:
This should be resolvable 🤔

Reply:
It can be resolved when running and using it, but linting can't resolve it here